docs/user_guide/vis/index.rst

```diff
@@ -1,88 +1,284 @@
+
+File actions
+    * |ifv-refresh| Refresh - Refresh with view with any changes made to the flowsheet in Python.
+      This also has the effect of saving the current layout. Changes in the units or their connections will of
+      course alter the layout.
+    * |ifv-save| Save - Save the current layout to the data store that was specified with the visualization
+      was launched. This does *not* update with any changes made to the flowsheet in Python. Neither does it
+      have any effect on the Python flowsheet values (the IFV can never modify the flowsheet).
+    * |ifv-export| Export - Save the flowsheet as a Scalable Vector Graphics (SVG) file, a common format for
+      images that consist of "vector" elements like boxes, lines, and text. SVG files can be viewed like images
+      by most programs that allow image viewing, and even edited with a program like `Inkscape <https://inkscape.org/>`_.
+    * Quit - Close the UI window.
+
+.. |ifv-refresh| image:: ../../_images/icons/refresh-24px.svg
+.. |ifv-save| image:: ../../_images/icons/save-24px.svg
+.. |ifv-export| image:: ../../_images/icons/export.svg
+
+.. _ifv-action-view:
+
+View actions
+    * Labels - Toggle view of the annotations, or labels
+    * Grid - Toggle a background "grid"
+    * Zoom - Zoom the view of the flowsheet within the canvas. This is a label for a set of related options.
+    * |ifv-zoom-in| Zoom in - Zoom in by 25%
+    * |ifv-zoom-out| Zoom out - Zoom out by 25%
+    * |ifv-zoom-reset| Reset - Reset zoom to 100%
+    * Canvas size - Change the size of the "canvas" on which the flowsheet is drawn. This lets you adapt
+      the IFV for different display (screen) sizes.
+
+.. |ifv-zoom-in| image:: ../../_images/icons/zoom_in-24px.svg
+.. |ifv-zoom-out| image:: ../../_images/icons/zoom_out-24px.svg
+.. |ifv-zoom-reset| image:: ../../_images/icons/zoom_out_map-24px.svg
+
+.. _ifv-action-help:
+
+|ifv-help| Help actions
+   * About - General information about the IFV
+
+.. TBD  * Documentation - Links to the online documentation
+
+.. |ifv-help| image:: ../../_images/icons/help_outline-24px.svg
+
+(1) Unmodified from
+`Google Material Design system icons <https://material.io/design/iconography/system-icons.html#design-principles>`_
+
+Graph
++++++
+
+The graph area is where you can rearrange the flowsheet as you need and zoom in on particular sections.
+The main visual components are called Units, Lines, and Annotations.
+
+Units
+    *Units* is the term used for any geometric shape in the flowsheet that is connected by lines.
+    The three types of units are IDAES unit models (such as a Flash, Mixer, or Splitter), inlets, or
+    outlets. Units can be moved by clicking and dragging them. If you double-click on a unit, it
+    will rotate 90 degrees.
+
+Lines
+    The lines connecting units, also called "arcs", can be manipulated by clicking and dragging.
+    If you double-click on a line, you will create a new segment that can be used for routing the line
```

Comment:
You can also get rid of a line segment by clicking on the dot when you hover over the line